extensions/ql-vscode/src/common/commands.ts

```diff
@@ -36,6 +36,7 @@ export type SingleSelectionCommandFunction<Item> = (
 // See https://code.visualstudio.com/api/references/commands
 export type BuiltInVsCodeCommands = {
   "markdown.showPreviewToSide": (uri: Uri) => Promise<void>;
+  setContext: (key: string, value: unknown) => Promise<void>;
```

Comment:
Should we limit the `key` to just `codeQL`? I don't think we have any use cases for using a non-CodeQL key, so something like this should work:

```suggestion
  setContext: (
    key: `${"codeql" | "codeQL"}${string}`,
    value: unknown,
  ) => Promise<void>;
```

Reply:
LGTM. All existing uses do indeed start with CodeQL and I agree we don't currently see a need to set other keys, so it would be good to avoid doing it accidentally.